qualys-web-application-scanner/info.json

```diff
@@ -1217,7 +1431,980 @@
+          "tooltip": "Set this to true to indicate whether the schedule is active."
+        },
+        {
+          "title": "Type",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "multiselect",
+          "options": [
+            "VULNERABILITY",
+            "DISCOVERY"
+          ],
+          "name": "type",
+          "description": "Specify the scheduled scan type. You can choose from the following options: VULNERABILITY or DISCOVERY."
+        },
+        {
+          "title": "Web App Name",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "webApp.name",
+          "description": "Specify the name of the web application being scanned.",
+          "tooltip": "Specify the name of the web application being scanned."
+        },
+        {
+          "title": "Web App ID",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "webApp.id",
+          "description": "Specify the ID of the web application being scanned.",
+          "tooltip": "Specify the ID of the web application being scanned."
+        },
+        {
+          "title": "Web App Tag ID",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "webApp.tags.id",
+          "description": "Specify the ID of the tag applied to the web application being scanned.",
+          "tooltip": "Specify the ID of the tag applied to the web application being scanned."
+        },
+        {
+          "title": "Last Scan Status",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "multiselect",
+          "options": [
+            "SUBMITTED",
+            "RUNNING",
+            "FINISHED",
+            "TIME_LIMIT_EXCEEDED",
+            "SCAN_NOT_LAUNCHED",
+            "SCANNER_NOT_AVAILABLE",
+            "ERROR",
+            "CANCELED"
+          ],
+          "name": "lastScan.status",
+          "description": "Specify the status of the last scan. You can choose from the following options: SUBMITTED, RUNNING, FINISHED, TIME_LIMIT_EXCEEDED, SCAN_NOT_LAUNCHED, SCANNER_NOT_AVAILABLE, ERROR or CANCELED"
+        },
+        {
+          "title": "Last Scan Launch Date",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "datetime",
+          "name": "lastScan.launchedDate",
+          "description": "Specify the date and time to return all the web application which was last scanned in WAS after specified date and time in UTC date/time format.",
+          "tooltip": "Specify the date and time to return all the web application which was last scanned in WAS after specified date and time in UTC date/time format."
+        },
+        {
+          "title": "Multi",
```

Comment:
Can you change the title to something more understandable

Reply:
The API Doc uses same name and tooltip is provided with the input parameter to make it clear.